evcouplings/mutate/protocol.py

```diff
@@ -14,7 +14,9 @@
 from evcouplings.mutate.calculations import (
     single_mutant_matrix, predict_mutation_table
 )
-import evcouplings
+from evcouplings.visualize.mutations import (
+    plot_mutation_matrix, mutation_pymol_script
```

Comment:
This is not blowing up with a circular import upon this change (which is why I had the ugly original solution I think)?

Reply:
Actually it was blowing up the way it was written, the only way to get rid of it was changing to what I have.... I can't explain it